packages/http/test/fixtures/httpResponses.fixture.ts

```diff
@@ -0,0 +1,19 @@
+
+import { VALUES } from './values';
+
+const HTTP_RESPONSES = new Map();
```

Comment:
We commonly use plain objects for tests, so it would be more consistent to use one here.